app/views/layouts/_banner.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if ENV['SHOW_MAINTENANCE_BANNER'] && ENV['SHOW_MAINTENANCE_BANNER'].to_s.downcase == "true" %>
```

Comment:
Think we should also make sure that `MAINTENANCE_BANNER_TEXT` isn't empty as well.